### ISSUE TITLE
Taxonomy: Advanced Browse

### ISSUE THREAD

**Issue by dlschwartz** [srophe/srophe-eXist-app#1212]
@wsalesky I'm afraid the advanced browse is broken too. We used to have a faceted browse here that listed all the categories on the right and allowed users to select multiple categories and find listed on the right the headwords that appeared in however many categories the user selected.

**Comment by dlschwartz** [comment]
Here's a picture of the old advanced browse: 
![Picture1](https://user-images.githubusercontent.com/4984270/74278704-4064fe80-4cdf-11ea-8520-5c548c2f1788.jpg)

**Comment by wsalesky** [comment]
@dlschwartz This should be fixed. How does it look now?

**Comment by dlschwartz** [comment]
Looks very good. Thank you.

**Comment by dlschwartz** [comment]
@wsalesky it looks like this is broken again. Thanks Winona.

**Comment by wsalesky** [comment]
@dlschwartz Can you clarify? What is broken?

**Comment by dlschwartz** [comment]
@wsalesky it looks like the same problem as above. The list of categories down the left side should include all of the passive values in, for example `<relation ref="skos:broadMatch" active="http://syriaca.org/keyword/abbasids" passive="http://syriaca.org/keyword/dynasties"/>`. Each should have a count of the number of keywords under that broader category. See the image here: https://github.com/srophe/srophe-eXist-app/issues/1212#issuecomment-584849223. 

The browse also used to be faceted so that when you selected a broader category on the left, all the keywords in that category displayed on the right. If any of those keywords were also in one or more different broader categories those broader categories would continue to appear on the left.

Thanks Winona.

**Comment by wsalesky** [comment]
@dlschwartz Is this an urgent fix? If not do you mind if we wait until we roll out the new version? At this point I am maintaining two very different code bases while we wait to deploy eXist v5, and the facets are completely different. If you are actively using dev I will fix it today, if not then it would be great to wait.

**Comment by dlschwartz** [comment]
@wsalesky absolutely, let's wait! I was hoping it would just take a minute to re-do whatever you did last time. I should have said that this was low priority because it definitely is. Thanks.
